src/sentry/static/sentry/app/utils/errorHandler.jsx

```diff
@@ -16,10 +12,20 @@ export default function errorHandler(Component) {
     }
 
     state = {
+      // we are explicit if an error has been thrown since errors thrown are not guaranteed
+      // to be truthy (e.g. throw null).
       hasError: false,
       error: null,
     };
 
+    componentDidCatch(error, info) {
+      // eslint-disable-next-line no-console
+      console.error(
```

Comment:
@billyvg looks like Sentry is outputting the stacktraces. 

Just for completeness, I've `console.error`'d the component stacktrace here. see: https://reactjs.org/docs/error-boundaries.html#component-stack-traces

Looks like that PR you created recently will help with this: https://github.com/getsentry/sentry/pull/13607 😄